### ISSUE TITLE
Issue with `addSortArg` with graphql version 0.12+

### ISSUE THREAD

**Issue by Anthonyzou** [graphql-compose/graphql-compose#104]
I will get a stack trace like this.

```
/Users/anthonyou/git/core-server/node_modules/graphql-compose/lib/resolver.js:560
      var existedIdx = sortEnumType._values.findIndex(function (o) {
                                            ^
TypeError: Cannot read property 'findIndex' of undefined
```

```typescript
FileTC.wrapResolver('findMany', newResolver => {
      return newResolver
        .addSortArg({
          name: 'NAME_ASC',
          description: '',
          value: { name: 1 } as any,
        })
})
```

It's probably related to this [line](https://github.com/nodkz/graphql-compose/blob/7328b1aaf2b00deecfc2e4b32905dbd209283853/src/resolver.js#L595). It should probably be using `.getValues()`?

**Comment by nodkz** [comment]
I'll update package in upcomming week. Please keep patience. There is more than 100 commits in graphql-js since last release. ;)

Thanks for you help 👌

**Comment by nodkz** [comment]
This problem solved in `next` branch. 
BUT it waiting for a new `graphql-js` version 0.12.4 (required changes already merged in its master, but not yet published on npm).

So `graphql-compose@2.11.0` will be compatible with `graphql-js@0.12.4`.

**Comment by Anthonyzou** [comment]
with version 4.0.0 using `addSortArg` on a resolver gives the error:

> Resolver should have `sort` arg with type GraphQLEnumType. But got: EnumTypeComposer

**Comment by nodkz** [comment]
@Anthonyzou thanks for reporting!
Should be fixed in 4.0.1

**Comment by Anthonyzou** [comment]
Thanks for the the quick fix!
